File: Modul_2/Bahasa Python/PRAK202-2510817120005-NorKhalisyah.py
a = float(input("Masukkan nilai pertama: "))
b = float(input("Masukkan nilai kedua: "))

hasil = a + b

if a == int(a):
    print(f'Hasil dari penjumlahan nilai pertama "{a:.0f}" dan nilai kedua "{b:.1f}" adalah "{hasil:.2f}"')
else:
    print(f'Hasil dari penjumlahan nilai pertama "{a:.2f}" dan nilai kedua "{b:.1f}" adalah "{hasil:.2f}"')

File: Modul_2/Bahasa C/PRAK202-2510817120005-NorKhalisyah.c
#include <stdio.h>
int main () {
    double a, b, hasil;

    printf("Masukkan nilai pertama: ");
    scanf("%lf", &a);
    
    printf("Masukkan nilai kedua: ");
    scanf("%lf", &b);

    hasil = a + b;

    if(a == (int) a)
    {
        printf("Hasil dari penjumlahan nilai pertama \"%.0f\" dan nilai kedua \"%.1f\" adalah \"%.2f\"\n", a, b, hasil);
    }
    else
    {
        printf("Hasil dari penjumlahan nilai pertama \"%.2f\" dan nilai kedua \"%.1f\" adalah \"%.2f\"\n", a, b, hasil);
    }
    return 0;
}